### ISSUE TITLE
Training is too slow

### ISSUE THREAD

**Issue by SergioQuijanoRey** [SergioQuijanoRey/TFG#17]
- I think it's because the computation of some metrics to log them in the middle of the process
- Either make the metrics computation run faster (writing them in pure pytorch / numpy) or use a smaller subset of the original dataset

**Comment by SergioQuijanoRey** [comment]
For now, in MNIST dataset #16 training is fast enough

Principal change was using some portion of training and validation set, instead of whole datasets
